Pipelines/Templates/run-canvas-tests.yml

```diff
@@ -0,0 +1,89 @@
+parameters:
+  - name: serviceConnection
+    type: string
+  - name: aadHost
+    type: string
+    default: login.microsoftonline.com
+  - name: solutionName
+    type: string
+  - name: testAccountUsername
+    type: string
+  - name: testAccountPassword
+    type: string
+  - name: makerPortalUrl
+    type: string
+    default: https://make.powerapps.com
+  - name: loginMethod
+    type: string
+    # default: CloudIdentity
+    # values:
+    #   - CloudIdentity # Uses the default Office 365 Sign-In Service for authentication
+    #   - FederatedIdentity # Uses a Custom Login Action method using AD FS 2.0 or other Security Token Services for authentication. Custom code is required.
+    #   - PassThrough # Federated login scenario where the provided user credential is automatically logged in, requires special agent configurations
+
+steps:
+- pwsh: |
+    # only attempt to run tests if a test username was provided as a parameter
```

Comment:
Should we expand on this check or use conditions on the task to only run tests on PRs similar to running Solution Checker using Branch policies? Don't want tests running in Production.